content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -880,44 +1045,131 @@ N/A
 
 In most cases, you want your database to create the ID. To do this, annotate the ID field with the `@default` attribute and initialize the field with a [function](#attribute-functions).
 
+##### Generate auto-incrementing integers as IDs
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases only" icon="database"/>]}>
+<tab>
+
+```prisma
+model User {
+  id    Int     @id @default(autoincrement())
+  name  String
+}
+```
+
+</tab>
+<tab>
+
+</tab>
+</TabbedContent>
+
+##### Generate `ObjectId` as IDs (MongoDB only)
+
+<TabbedContent tabs={[<FileWithIcon text="MongoDB only" icon="database"/>]}>
+<tab>
+
+```prisma
+model User {
+  id    String     @id @map("_id") @db.ObjectId @default(dbgenerated())
+  name  String
+}
+```
+
+</tab>
+<tab>
+
+</tab>
+</TabbedContent>
+
 ##### Generate `cuid()` values as IDs
 
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
 ```prisma
 model User {
   id    String  @id @default(cuid())
   name  String
 }
 ```
 
+</tab>
+<tab>
+
+```prisma
+model User {
+  id    String  @id @default(cuid()) @map("_id")
+  name  String
+}
+```
+
+Do not attempt to use `cuid()` to generate an `ObjectId` - this requires specific syntax.
+
+</tab>
+</TabbedContent>
+
 ##### Generate `uuid()` values as IDs
 
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+
+<tab>
+
 ```prisma
 model User {
   id    String  @id @default(uuid())
   name  String
 }
 ```
 
-##### Generate auto-incrementing integers as IDs
+</tab>
+
+<tab>
 
 ```prisma
 model User {
-  id    Int     @id @default(autoincrement())
+  id    String  @id @default(uuid()) @map("_id")
   name  String
 }
 ```
 
+Do not attempt to use `uuid()` to generate an `ObjectId` - this requires specific syntax.
```

Comment:
Should we maybe just not show this instead and link to `autogenerated` here?

Reply:
Hm.. I'm not sure I understand what you mean. You can use `cuid()` and `uuid()` for MongoDB IDs as long as those IDs are not an `ObjectId`. Changed to this on both functions:

```
The ID field in this example is specifically _not_ an `ObjectId` (the `id` field does not define the `@db.ObjectId` attribute). You cannot use `cuid()` to generate an `ObjectId` - you must use `dbgenerated()` instead.
```

Reply:
Ah ok - that situation is still confusing to me.